### ISSUE TITLE
Request for upcoming pack

### ISSUE THREAD

**Issue by goreae** [sinkillerj/ProjectE#471]
You know how in EE3 if you had Redpower installed you could boil down tungsten ore to 2 diamonds worth of EMC, but the ore doesn't show up in the tablet for transmuting? That's what I want. I want to make a pack centered on Project E, but only make raw resources transmutable. So you can take iron out of the tablet to make a chestplate, but you can't pull out a chestplate itself. However, I would still want to be able to turn the chestplate back into 8 iron's worth of EMC, without teaching it to the tablet, or being able to make it with the condenser. I want my players to use EE3, but not so much that they can pull a nuclear reactor out of the tablet whenever they want. But if they make a reactor, but need the EMC for something else, they can burn up the reactor and transmute new materials from that.

**Comment by Nova001Sol** [comment]
You mean like a blacklist or a whitelist for creation?

**Comment by MaPePeR** [comment]
I think what you want there is buy and sell prices that differ. This is not planned at the moment.

Also you are talking a lot about EE3 in your post... ProjectE is about EE2...

**Comment by goreae** [comment]
Yeah, that was just because I'm so used to saying EE instead of projectE or PE... Just general derpiness on my part. I just edited the post to fix it.

And I don't know how a differing price would be useful at all, but a creation blacklist would be perfect. That sounds like exactly what I want. Just a list of items that can't be created. That would be really useful. But differing creation/destruction prices? That doesn't sound to equivalent.

**Comment by shobu9** [comment]
@gorea I think you can set items Emc and Etc in the config 
This is what i understand you want...

**Comment by goreae** [comment]
No, I know you can set their EMC, I just want a creation blacklist. A list of stuff that cannot be created with EMC, but can still be broken down into EMC.
